Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -220,10 +251,80 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
 
         let fontVariationPropertiesSection = new WI.DetailsSection("font-variation-properties", WI.UIString("Variation Properties", "Variation Properties @ Font Details Sidebar Section", "Section title for font variation properties."), [this._fontVariationsGroup]);
         this.element.appendChild(fontVariationPropertiesSection.element);
+
+        // Custom Fonts Properties
+        this._createDataGrid();
+        this._customFontsRow = new WI.DetailsSectionDataGridRow(null, WI.UIString("No Custom Fonts", "No Custom Fonts @ Font Details Sidebar Property", "Message shown when there are no custom fonts."));
+        this._customFontsRow.dataGrid = this._dataGrid;
+
+        this._customFontsGroup = new WI.DetailsSectionGroup([this._customFontsRow]);
+
+        let customFontsSection = new WI.DetailsSection("custom-fonts-properties", WI.UIString("Custom Fonts", "Custom Fonts @ Font Details Sidebar Section", "Section title for all custom fonts."), [this._customFontsGroup], null, true);
+
+        this.element.appendChild(customFontsSection.element);
     }
 
     // Private
 
+    _createDataGrid()
+    {
+        // All Fonts Properties
+        let fontColumns = {
```

Comment:
Style: this can be `const` since it doesn't change between Web Inspector sessions